OrbitGl/GpuTrack.cpp

```diff
@@ -351,3 +351,20 @@ std::vector<std::shared_ptr<TimerChain>> GpuTrack::GetAllChains() {
   }
   return chains;
 }
+
+//-----------------------------------------------------------------------------
+void GpuTrack::SetTimeline(std::string_view timeline) {
```

Comment:
nit. I find "timeline" to be confusing to describe a label or a name.

Reply:
I kind of agree but it's also my first time looking at that code. And I don't have a good suggestion for a name, so I personally would "fix" that problem by avoiding that function all the way. I would define a free function that contains the absl::StrFormat-logic from below and maps from a gpu-track name to a gpu-track label. (Please feel free to ignore that.)

Reply:
Yes, it is. "Timeline" is the Linux kernel name for this, for example, used in the tracepoint format. 

I will follow @beckerhe 's suggestion to map the timeline to the display name.

Reply:
Done.